server/historian/packages/historian-base/package.json

```diff
@@ -24,13 +24,14 @@
   },
   "dependencies": {
     "@fluidframework/common-utils": "^0.26.0",
-    "@fluidframework/gitresources": "^0.1017.1",
-    "@fluidframework/protocol-definitions": "^0.1017.1",
-    "@fluidframework/server-services": "^0.1017.1",
-    "@fluidframework/server-services-client": "^0.1017.1",
-    "@fluidframework/server-services-core": "^0.1017.1",
-    "@fluidframework/server-services-utils": "^0.1017.1",
-    "@fluidframework/server-test-utils": "^0.1017.1",
+    "@fluidframework/gitresources": "^0.1020.0-15688",
+    "@fluidframework/protocol-definitions": "^0.1020.0-15688",
+    "@fluidframework/server-services": "^0.1020.0-15688",
+    "@fluidframework/server-services-client": "^0.1020.0-15688",
+    "@fluidframework/server-services-core": "^0.1020.0-15688",
+    "@fluidframework/server-services-utils": "^0.1020.0-15688",
+    "@fluidframework/server-test-utils": "^0.1020.0-15688",
```

Comment:
There was recently a PR ( #4897 ) to remove pre-release dependencies across the project. There was no description, though, so it's hard to say why. @wes-carlson?

Reply:
Not sure why prerelease dependencies are removed. But the reason I use `^0.1020.0-15688` instead of `^0.1020.0` is `0.1020.0` is not published.

Reply:
Changed to ^0.1019.0